### ISSUE TITLE
Invalid token issue in terracom website

### ISSUE THREAD

**Issue by shivakrishna-vcare** [shivakrishna-vcare/OrangeScrum#191]

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>ending www.terracomwireless.com/prepaid.php<br />Transmitting file data .done<br />Committing transaction...<br />Committed revision 79775.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>79775 &gt; QA 79776 &gt; Demo 79777</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>&nbsp;</p>
<p>79775 &gt; QA 79776 &gt; Demo 79777 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>79775 &gt; QA 79776 &gt; Demo 79777 &gt; Production 79778<br />&nbsp;<br />&nbsp;<br />&nbsp;<br />&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/address_ajax.phpTransmitting file data .doneCommitting transaction...Committed revision 80354.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>80354 &gt; QA 80356 &gt; DEMO 80357</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Irshad Mohd commented:<br/><p>Please move below to live.&nbsp;</p>
<p>80354 &gt; QA 80356 &gt; Demo 80357 &gt; Live</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>&nbsp;</p>
<p>80354 &gt; QA 80356 &gt; Demo 80357 &gt; Production</p>
<p>&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>80354 &gt; QA 80356 &gt; Demo 80357 &gt; Production 80366</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa, demo and live</p>
<p>Sending www.terracomwireless.com/prepaid.phpTransmitting file data .doneCommitting transaction...Committed revision 80396.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>80396 &gt; QA 80399 &gt; Demo 80400 &gt; Live 80401</p>
<p>&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/thanks.phpTransmitting file data .doneCommitting transaction...Committed revision 81269.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81269 &gt; QA 81270 &gt; DEMO 81271</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Irshad Mohd commented:<br/><p>Now approved spelling mistake fixed.&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>81269 &gt; QA 81270 &gt; DEMO 81271 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81269 &gt; QA 81270 &gt; DEMO 81271 &gt; Production 81278</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/auto_pay_list.phpSending www.terracomwireless.com/plan_detail.phpSending www.terracomwireless.com/user_account.phpTransmitting file data ...doneCommitting transaction...Committed revision 81286.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81286 &gt; QA 81287 &gt; DEMO 81288</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pavan.kumar commented:<br/><p>Pushkar sir,<br /><br />change the label 'Get CDR' label to 'Get Call Detail Records' on website,it is working fine on Demo move to next label.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/thanks.phpTransmitting file data .doneCommitting transaction...Committed revision 81292.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81292 &gt; QA 81293 &gt; DEMO 81294</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/GetAuthenticationCode.phpSending www.terracomwireless.com/thanks.phpTransmitting file data ..doneCommitting transaction...Committed revision 81298.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/thanks.phpTransmitting file data .doneCommitting transaction...Committed revision 81300.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81298 &gt; QA 81301 &gt; DEMO 81302</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/thanks.phpTransmitting file data .doneCommitting transaction...Committed revision 81303.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81303 &gt; QA 81306 &gt; DEMO 81307</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>81286 &gt; QA 81287 &gt; DEMO 81288 &gt; Production<br />81298 &gt; QA 81301 &gt; DEMO 81302 &gt; Production</p>
<p>81303 &gt; QA 81306 &gt; DEMO 81307 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81286 &gt; QA 81287 &gt; DEMO 81288 &gt; Production 81322<br />81298 &gt; QA 81301 &gt; DEMO 81302 &gt; Production 81324</p>
<p>81303 &gt; QA 81306 &gt; DEMO 81307 &gt; Production updated<br />&nbsp;<br />&nbsp;<br />&nbsp;<br />&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/cdr_new.phpTransmitting file data .doneCommitting transaction...Committed revision 81330.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81330 &gt; QA 81332 &gt; Demo 81333</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/cdr_new.phpTransmitting file data .doneCommitting transaction...Committed revision 81334.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81334 &gt; QA 81336 &gt; DEMO 81337</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>please move below revision to qa and demo&nbsp;</p>
<p>Sending www.terracomwireless.com/cdr_new.phpTransmitting file data .doneCommitting transaction...Committed revision 81342.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81342 &gt; QA 81343 &gt; DEMO 81344</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>81342 &gt; QA 81343 &gt; DEMO 81344 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>81342 &gt; QA 81343 &gt; DEMO 81344 &gt; Production 81349</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/includes/otherheader.inc.phpTransmitting file data .doneCommitting transaction...Committed revision 82328.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>82328 &gt; QA 82330 &gt; DEMO 82332</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/add_data_internet.phpTransmitting file data .doneCommitting transaction...Committed revision 83492.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>83492 &gt; QA 83493 &gt; DEMO 83494</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/add_data_internet.phpSending www.terracomwireless.com/auto_pay.phpSending www.terracomwireless.com/auto_pay_list.phpSending www.terracomwireless.com/login.phpSending www.terracomwireless.com/make-a-payment.phpSending www.terracomwireless.com/more-minutes.phpSending www.terracomwireless.com/renew_plan.phpSending www.terracomwireless.com/step4_new.phpSending www.terracomwireless.com/upgrade_phone_payment.phpSending www.terracomwireless.com/upgrades.phpTransmitting file data ..........doneCommitting transaction...Committed revision 83568.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>83568 &gt; QA 83578 &gt; DEMO 83579</p>
<p>&nbsp;</p>
<p>&nbsp;</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/check-balance-api-call.phpTransmitting file data .doneCommitting transaction...Committed revision 86314.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>86314 &gt; QA 86317 &gt; DEMO 86319</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>&nbsp;</p>
<p>86314 &gt; QA 86317 &gt; DEMO 86319 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>86314 &gt; QA 86317 &gt; DEMO 86319 &gt; Production 86325</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>Please move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/step1.phpTransmitting file data .doneCommitting transaction...Committed revision 88403.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>88403 &gt; QA 88406 &gt; DEMO 88408</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;neeraj.kumar commented:<br/><p>PLease move below revision to qa and demo</p>
<p>Sending www.terracomwireless.com/step1.phpTransmitting file data .doneCommitting transaction...Committed revision 88429.</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>88429 &gt; QA 88431 &gt; DEMO 88433</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Irshad Mohd commented:<br/><p>Please move below to live.&nbsp;</p>
<p>88429 &gt; QA 88431 &gt; Demo 88433 &gt; Live</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;pushkar.kumar commented:<br/><p>&nbsp;</p>
<p>88429 &gt; QA 88431 &gt; Demo 88433 &gt; Production</p>

**Comment by shivakrishna-vcare** [comment]
&#10148; &nbsp;Linux_user Team commented:<br/><p>88429 &gt; QA 88431 &gt; Demo 88433 &gt; Production 88441</p>
